server/schedulers/hot_region.go

```diff
@@ -113,11 +111,9 @@ type hotScheduler struct {
 	regionPendings map[uint64][2]*operator.Operator
 
 	// temporary states but exported to API or metrics
-	stats           *storeStatistics // store id -> hot regions statistics
+	stLoadInfos     *storeLoadInfos
```

Comment:
Maybe `storeLoadInfos` is more friendly?

Reply:
That will be the same as its type. I think `stLoadInfos` is fine.